frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.unit.spec.tsx

```diff
@@ -0,0 +1,119 @@
+import { renderWithProviders, screen } from "__support__/ui";
+import type { EmbeddingEntityType } from "metabase/embedding-sdk/store";
+import * as domUtils from "metabase/lib/dom";
+import {
+  createMockEmbedOptions,
+  createMockEmbedState,
+  createMockState,
+} from "metabase-types/store/mocks";
+
+import { BrowseNavSection } from "./BrowseNavSection";
+
+describe("BrowseNavSection", () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("should render Models, Databases, and Metrics when not embedding in iframe", () => {
+    setup();
+
+    expect(
+      screen.getByRole("listitem", { name: "Browse models" }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole("listitem", { name: "Browse databases" }),
+    ).toBeInTheDocument();
```

Comment:
ah, this is because of the condition that we only show databases if `table` is present? 

```ts
     {hasDataAccess &&
        (!isEmbeddingIframe || entityTypes.includes("table")) && (
```

Reply:
well, the test you're commenting on is the case where we're not embedding, aka just Metabase. We'd show 3 things
- Models
- Database: if having necessary permissions which is simulated by passing `hasDataAccess`
- Metrics

In embedding iframe, it's as you've commented.

tl;dr
if not embedding iframe: show this if `hasDataAccess`
if embedding show this if `hasDataAccess` and `entityTypes` contains `table`

I hope this makes sense.

Reply:
got it! sorry, I commented on the wrong line - all good then